src/OrleansServiceBus/Providers/Streams/EventHub/EventHubAdapterReceiver.cs

```diff
@@ -202,7 +219,12 @@ public Task MessagesDeliveredAsync(IList<IBatchContainer> messages)
 
                 // clear cache and receiver
                 IEventHubQueueCache localCache = Interlocked.Exchange(ref cache, null);
-                EventHubReceiver localReceiver = Interlocked.Exchange(ref receiver, null);
+#if NETSTANDARD
+                PartitionReceiver
+#else
+                EventHubReceiver  
+#endif
```

Comment:
I always got someone going against `var` whenever I use it on my PRs :) Will change that.